--- error.tpl	Tue Dec 30 19:09:51 2003
+++ tiki.tpl	Sat Oct  4 18:53:44 2003
@@ -29,7 +29,7 @@
 		  {if $feature_usermenu eq 'y'}
 		  	::&nbsp;<a class="tikitopmenu" href="tiki-usermenu.php">{tr}Favorites{/tr}</a>
 		  {/if}
- 	      {if $feature_tasks eq 'y' and $tiki_p_tasks eq 'y'}
+ 	          {if $feature_tasks eq 'y' and $tiki_p_tasks eq 'y'}
 		  	::&nbsp;<a class="tikitopmenu" href="tiki-user_tasks.php">{tr}Tasks{/tr}</a>
 		  {/if}
 		  {if $feature_user_bookmarks eq 'y' and $tiki_p_create_bookmarks eq 'y'}
@@ -71,32 +71,30 @@
     <tr>
       {if $feature_left_column eq 'y'}
       <td id="leftcolumn">
+      <div id="leftcolumnheader"></div>
+
       {section name=homeix loop=$left_modules}
       {$left_modules[homeix].data}
       {/section}
       
       </td>
       {/if}
-      <td id="centercolumn"><div id="tiki-center">
-      <br /> 
+      <td id="centercolumn">
+      <div id="centercolumnheader"></div>
+      <div id="tiki-center">
         <div class="cbox">
         <div class="cbox-title">
         {tr}Error{/tr}
         </div>
         <div class="cbox-data">
         {$msg}<br /><br />
         <a href="javascript:history.back()" class="linkmenu">{tr}Go back{/tr}</a><br /><br />
         <a href="{$tikiIndex}" class="linkmenu">{tr}Return to home page{/tr}</a>
         </div>
         </div>
       </div>
       </td>
       {if $feature_right_column eq 'y'}
       <td id="rightcolumn">
+      <div id="rightcolumnheader"><div id="rchleft"></div></div>
+
+      <div id="rightcolumnbody">
+      <div width="100%" align="right"><img src="styles/railfuture/rfboxlogo.gif" width="140" height="21" alt="Railfuture"/></div>
       {section name=homeix loop=$right_modules}
       {$right_modules[homeix].data}
       {/section}
-      
+      </div>
       </td>
       {/if}
     </tr>
